ivy/functional/backends/jax/layers.py

```diff
@@ -60,5 +60,15 @@ def conv3d(*_):
     raise Exception("Convolutions not yet implemented for jax library")
 
 
-def conv3d_transpose(*_):
-    raise Exception("Convolutions not yet implemented for jax library")
+def conv3d_transpose(
+    x: JaxArray,
+    filters: JaxArray,
+    strides: Union[int, Tuple[int], Tuple[int, int], Tuple[int, int, int]],
+    padding: str,
+    data_format: str = "NDHWC",
+    dilations: int = 1,
+) -> JaxArray:
+    return JaxArray.nn.conv3d_transpose(
```

Comment:
this seems to be invalid in Jax, maybe you can try implementing this function through jax.lax.convd_transpose? Reference here: https://jax.readthedocs.io/en/latest/_autosummary/jax.lax.conv_transpose.html 

Some additional code may be needed as well, you might want to check out the its implementation in mxnet or torch for some insights!

Reply:
Just a quick question if you don't mind. 
I just need to make corrections to line 68-74, right?

And also, do I need to use all the parameters mentioned in the documentation?

Reply:
yep, just the function body

and the parameters used should be the ones common to the arguments here, eg. strides, padding, dilation, etc.

Reply:
Alright I just made changes to Jax. 
how do I commit it to this pull request?😅